src/wiktextract/import_utils.py

```diff
@@ -5,14 +5,11 @@
 
 def import_extractor_module(
     lang_code: str, module_name: str
-) -> types.ModuleType:
-    default_module_name = f"wiktextract.extractor.en.{module_name}"
+) -> types.ModuleType | None:
```

Comment:
I've added --edition and --dump-file-lang-code to the regen script because they were missing for the main En invocation, hopefully things work out.

Reply:
The default en module is removed at here is because I think load en edition code for not implemented edition is inappropriate, return `None` and raise exception in the caller code is fine.

Reply:
Yeah, it would have just broken kaikki's regen for a while if I hadn't thought about it. Should be fine.